react/components/Dropdown/index.js

```diff
@@ -103,6 +103,9 @@ class Dropdown extends Component {
       selectTestId,
     } = this.props
 
+    const valueLabel = this.getValueLabel()
+    const showCaption = !valueLabel
+
```

Comment:
this is already fixed (https://github.com/vtex/styleguide/pull/1095). 🤔

Reply:
Nice! Didn't see it

Reply:
No prob. Just to keep you warn while reabasing 😁